### PR TITLE
Make 3d net layers work

### PR DESCRIPTION
**Pull request by mdecea**
Implements meshing of components with net layers in 3D for gdsfactory8

@simbilod

### CODE REVIEW

File: gplugins/common/utils/get_component_with_net_layers.py [hunk 1]
```diff
@@ -67,17 +66,23 @@ def get_component_with_net_layers(
...
+        )
+        net_component = net_component.remove_layers(layers=(port.layer,))
         for polygon in polygons:
             # If polygon belongs to port, create a unique new layer, and add the polygon to it
-
-            if gdstk.inside(
-                [port.center],
-                gdstk.offset(gdstk.Polygon(polygon), gf.get_active_pdk().grid_size),
-            )[0]:
+            if polygon.sized(0.005).inside(port.center):
```
Comment (simbilod) [inline gplugins/common/utils/get_component_with_net_layers.py:77]: do we still have a grid size we can use here? Is hardcoding this value safe?

File: gplugins/gmsh/parse_gds.py [hunk 2]
```diff
@@ -37,7 +36,7 @@ def fuse_polygons(component, layer, round_tol=4, simplify_tol=1e-4, offset_tol=N
             exterior_points.append((point.x / 1000, point.y / 1000))
         for hole_index in range(klayout_polygon.holes()):
             holes_points = []
-            for point in layer_region.each_polygon_hole(hole_index):
+            for point in klayout_polygon.each_point_hole(hole_index):
```
Comment (simbilod) [inline gplugins/gmsh/parse_gds.py:39]: nice catch

File: gplugins/common/utils/get_component_with_net_layers.py (hunk 1 above)
Comment (mdecea) [reply gplugins/common/utils/get_component_with_net_layers.py]: Fixed it to use the dbu

**Comment by simbilod** [comment]
LGTM now!

**Review by simbilod — APPROVED** [review]

**Pull request merged**